### ISSUE TITLE
New SQL runner miscellaneous UI/UX fixes

### ISSUE THREAD

**Issue by ZeRego** [lightdash/lightdash#10616]
Tracking small fixes while working on this milestone:
- [x] #10726
- [x] #10779
- [ ] #10829
- [x] #10683
- [ ] #10935
- [x] #10686
- [x] #10687
- [x] add default SQL. eg `SELECT * FROM lightdash-analytics.prod.github_daily LIMIT 25`
- [x] #10789
- [x] #10679
- [x] fix/remove double borders between panels 
- [x] ~reduce left sidebar padding (nice-to-have)~
- [x] #10781
- [x] debounce label field overrides and/or look into unblocking the UI thread when this happens
- [x] #10690
- [x] add content panel empty state
- [x] #10738
- [x] Add quoting when coping a table (e.g. when I click `pages` it should copy `` `database`.`table`.`pages` ``
- [x] It is not obvious to me that you can drag the results panel. I was stuck for a while with only full screen sql or full screen chart, I couldn't see both
- [x] adjust height of sql editor. the editor can go behind the `charts / results` panel, which means the last lines on the sql are hidden. 
- [x] can save charts in dashboards
- [x] #10783
- [x] ~Add autocomplete for fields. We can populate when a user selects a table and gets a list of fields. To get **all** fields it probably requires an extra endpoint~
- [ ] #10927
- [x] If I click on a sql in a dashboard, it takes me to the sql-runner-new but with a uuid instead of a slug, so it doesn't load correctly.
- [x] If I open this sql chart as a viewer, and try to run it, I get a backend error (403) as expected, but no feedback on the frontend.
- [x] #10844
- [ ] #10936
- [ ] #10937

Open a separate ticket/PR for each that requires multiple file/lines changes.

**Comment by Faisal-imtiyaz123** [comment]
@ZeRego can I work on this?

**Comment by ZeRego** [comment]
Hi @Faisal-imtiyaz123 👋 thanks for your interest in Lightdash. Tickets from this milestone are not open to external contributions as we are actively working on this milestone.
